Makefile

```diff
@@ -26,6 +26,10 @@ gofmt:
 check-vendor:
 	./scripts/check-vendor.sh
 
+.PHONY: validate-vendor-licenses
+validate-vendor-licenses:
+	go get github.com/frapposelli/wwhrd
```

Comment:
this should be part of `goget-tools` target

Reply:
Ack